devops/scripts/benchmarks/main.py

```diff
@@ -171,6 +172,7 @@ def main(directory, additional_env_vars, save_name, compare_names, filter):
         LlamaCppBench(directory),
         UMFSuite(directory),
         GromacsBench(directory),
+        OneDnnBench(directory),
```

Comment:
I think the list of benches in readme could be extended: https://github.com/intel/llvm/blob/sycl/devops/scripts/benchmarks/README.md#benchmarks